pyverbs/cmid.pyx

```diff
@@ -20,7 +21,7 @@ from pyverbs.cq cimport WC
 cdef class ConnParam(PyverbsObject):
 
     def __init__(self, resources=1, depth=1, flow_control=0, retry=5,
-                 rnr_retry=5, srq=0, qp_num=0):
+                 rnr_retry=5, srq=0, qp_num=0, data_len = 0):
```

Comment:
data_len=0
(according to PEP-8 convention - no spaces around '=' here)

Reply:
@EdwardSro Done.